### PR TITLE
docs: align eng/chs README for JIRA

### PR DESCRIPTION
**Pull request by klesh**
# Summary

- Update Chinese README to match up english version
- Small adjustment for English version

### Key Points

- [ ] This is a breaking change
- [x] New or existing documentation is updated

### CODE REVIEW

File: plugins/jira/README-zh-CN.md [hunk 1]
```diff
@@ -134,3 +91,148 @@ curl -XPOST 'localhost:8080/task' \
 
 ![Screen Shot 2021-08-13 at 10 07 19 AM](https://user-images.githubusercontent.com/27032263/129363083-df0afa18-e147-4612-baf9-d284a8bb7a59.png)
 
+## API
+
+### 数据源(Connection) 管理
```
Comment (Startrekzky) [inline plugins/jira/README-zh-CN.md:96]: Is this section empty?

File: plugins/jira/README.md [hunk 2]
```diff
@@ -23,16 +23,16 @@ Incident Count per 1k Lines of Code | Amount of incidents per 1000 lines of code
 
 ## Configuration
 
-In order to fully use this plugin, you will need to set various configurations via Dev Lake's `config-ui` service. Open `config-ui` on browser, by default the URL is http://localhost:4000, then go to **Data Integrations / JIRA** page. Here you can **Add** new connection to your JIRA connection or **Update** the settings if needed.
+In order to fully use this plugin, you will need to set various configurations via Dev Lake's `config-ui` service. Open `config-ui` on browser, by default the URL is http://localhost:4000, then go to **Data Integrations / JIRA** page. JIRA plugin currently supports multiple data source, Here you can **Add** new connection to your JIRA connection or **Update** the settings if needed.
```
Comment (Startrekzky) [inline plugins/jira/README.md:26]: `JIRA plugin currently supports multiple data source, Here you can **Add** new connection to your JIRA connection or **Update** the settings if needed.`
1. multiple data source, -> multiple data sources.
2. **Add** -> **add**
3. **Update** -> **update**

File: plugins/jira/README.md [hunk 3]
```diff
@@ -23,16 +23,16 @@ Incident Count per 1k Lines of Code | Amount of incidents per 1000 lines of code
...
-- Issue Type Mapping: JIRA is highly customizable, each JIRA instance may have a different set of issue types than others. In order to compute and visualize metrics for different instances, you need to map your issue types to standard ones. See [Issue Type Mapping](#issue-type-mapping) for detail 
-- Epic Key: unfortunately, epic relationship implementation in JIRA is based on `custom field`, which is vary from instance to instance. Please see [Find Out Custom Fields](#find-out-custom-fields) for the value
-- Story Point Field: same as Epic Key, lease see [Find Out Custom Field](#find-out-custom-field) for the value
+- Issue Type Mapping: JIRA is highly customizable, each JIRA instance may have a different set of issue types than others. In order to compute and visualize metrics for different instances, you need to map your issue types to standard ones. See [Issue Type Mapping](#issue-type-mapping) for detail.
+- Epic Key: unfortunately, epic relationship implementation in JIRA is based on `custom field`, which is vary from instance to instance. Please see [Find Out Custom Fields](#find-out-custom-fields).
```
Comment (Startrekzky) [inline plugins/jira/README.md:34]: [Find Out Custom Fields](#find-out-custom-fields) -> [Find Out Custom Fields](#find-out-custom-fields) for the value.

We can either keep 'for the value' or get rid of it, but the description of the 3 fields should be consistent.

**Review by Startrekzky — CHANGES_REQUESTED** [review]

File: plugins/jira/README-zh-CN.md (hunk 1 above)
Comment (klesh) [reply plugins/jira/README-zh-CN.md:96]: No, github review page shows only diff. you can click on top right corn of the file and click on 'view file' to see the full content.

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**